tests/page/page-focus.spec.ts

```diff
@@ -117,6 +117,69 @@ it('clicking checkbox should activate it', async ({ page, browserName, headless,
   expect(nodeName).toBe('INPUT');
 });
 
+it('tab should cycle between single input and browser', {
+  annotation: { type: 'issue', description: 'https://github.com/microsoft/playwright/issues/32339' }
+}, async ({ page, browserName, headless }) => {
+  it.fixme(browserName === 'chromium' && !headless, 'Chromium in headful mode keeps input focused.');
```

Comment:
try to run the test with `PLAYWRIGHT_CHROMIUM_USE_HEADLESS_NEW=1` - might behave differently.

Reply:
Yes, headless new also fails.